content/root/tools.textile

```diff
@@ -0,0 +1,65 @@
+---
+title: Tools
+meta_description: "A list of Ablly tools for increasingly productivity."
+meta_keywords: "tools, automation, productivity, developers, tooling, debugging"
+section: root
+index: 15
+jump_to:
+  Help with:
+    - VSCode#vscode-plugin
+    - Terraform Provider#terraform-provider
+    - GitHub actions#github-actions
+    - Developer Console#developer-console
+---
+
+This topic list some developer resources and tools you can use to increase your productivity with Ably.
+
+h2(#vscode-plugin). Ably for Visual Studio Code
+
+Use the Ably for Visual Studio Code extension to manage your Ably apps directly in Visual Studio Code.
+
+The extension enables you to:
+
+* List all apps in your Ably account.
+** View API keys, queues and integration rules for each app.
+** Copy API keys to your clipboard.
+* Create an Ably app.
+
+"Download here.":https://github.com/ably-labs/vscode-ably
+
+h2(#terraform-provider). Ably Terraform provider
+
+Terraform provider for Ably enables you to manage your Ably account using Infrastructure-as-Code (IaC), including managing your applications, namespaces, keys, queues, rules and more.
+
+"Download here.":https://github.com/ably/terraform-provider-ably
+
+h2(#github-actions). Ably Control API GitHub actions
+
+A GitHub Action to use the "Ably Control API":https://ably.com/docs/control-api. You can use this action to:
+
+* Create an Ably application.
+* Add an API key to an application, with a list of capabilities.
+
+"Download here.":https://github.com/ably-labs/ably-control-api-action
+
+h2(#developer-console). Developer console
+
+The "Ably dashboard":ably.com/dashboard contains a developer console. In the developer console you can view connection events. The following screenshot ilustrates an example connection:
+
+<a href="/images/screenshots/developer-console.png" target="_blank">
+  <img src="/images/screenshots/developer-console.png" style="width: 100%" alt="Developer Console">
+</a>
+
+You can also attach to a specific channel to view messages. You can also publish messages directly in the console:
+
+<a href="/images/screenshots/developer-console-channels.png" target="_blank">
+  <img src="/images/screenshots/developer-console-channels.png" style="width: 100%" alt="Developer Console - Channels">
+</a>
+
+The developer console also provides you with some REST API Curl command snippets that can help you test out Ably quickly:
```

Comment:
Worth emphasising that the credentials are pre-filled which is a big part of making them helpful?